src/sentry/static/sentry/app/views/groupDetails/shared/header.jsx

```diff
@@ -91,6 +91,18 @@ const GroupHeader = createReactClass({
     }
   },
 
+  searchTermWithoutQuery() {
+    const {location} = this.context;
+
+    const searchTermWithoutQuery = qs.stringify(omit(location.query, 'query'));
+
+    if (searchTermWithoutQuery.length <= 0) {
+      return '';
+    }
```

Comment:
I think we can also just pass an object to `Link` eg. `{pathname: '.../events/', query: omit(location.query, 'query'}` which would automatically handle this for us

Reply:
I've delegated the query string serialization to react-router.